elide-datastore/elide-datastore-multiplex/src/main/java/com/yahoo/elide/datastores/multiplex/MultiplexTransaction.java

```diff
@@ -114,7 +113,7 @@ public void close() throws IOException {
     }
 
     protected DataStoreTransaction getTransaction(Object object) {
-        return getTransaction(object.getClass());
+        return getTransaction(ClassType.of(object.getClass()));
```

Comment:
@aklish this because an endless loop when
```java
protected DataStoreTransaction getTransaction(Class<?> cls) {
```
became
```
protected DataStoreTransaction getTransaction(Type<?> cls) {
```